src/util/http_utils.js

```diff
@@ -591,6 +591,20 @@ function parse_content_length(req, options) {
     return size;
 }
 
+function authorize_session_token(req, options) {
+    const jwt_utils = require('./jwt_utils'); // eslint-disable-line global-require
```

Comment:
yes, jwt_utils.js checks for JWT_SECRET env var, and we do not pass it to the default-backing-store pod, which causes a panic ```PANIC: process uncaughtException Error: JWT_SECRET env variable not found. it must exist when running in kuberentes```
since authorize_session_token() will be called only from the endpoint pod I thought that requiring from inside the function is an ok handling, let me know what you think